src/controllers/user.js

```diff
@@ -74,6 +74,51 @@ const UserController = (userModel, authService, googleAPIService) => {
     });
   });
 
+  router.post('/hours', async (req, res)=>{
+    if (!req.headers) return res.status(400).json({
+        message: "Malformed Request"
+    });
+
+    if (!req.body) return res.status(400).json({
+      "message": "Malformed Request",
```

Comment:
Please ensure these two have the same format (one message uses quotes, the other doesnt